packages/oui-tile/src/button/tile-button.html

```diff
@@ -5,13 +5,19 @@
     ng-click="$ctrl.onClick()"
     ng-attr-aria-label="{{ :: $ctrl.ariaLabel }}">
     <span ng-transclude>{{ :: $ctrl.text }}</span>
-    <i class="oui-icon oui-icon-chevron-right" aria-hidden="true"></i>
+    <span class="oui-icon oui-icon-chevron-right" aria-hidden="true"></span>
 </button>
 <a
     ng-if="::!!$ctrl.href"
     class="oui-tile__button oui-button oui-button_link oui-button_icon-right oui-button_full-width"
     ng-href="{{:: $ctrl.href }}"
+    ng-attr-rel="{{:: $ctrl.external ? 'noopener' : undefined }}"
+    ng-attr-target="{{:: $ctrl.external ? '_blank' : undefined }}"
     ng-attr-aria-label="{{ :: $ctrl.ariaLabel }}">
     <span ng-transclude>{{ :: $ctrl.text }}</span>
-    <i class="oui-icon oui-icon-chevron-right" aria-hidden="true"></i>
+    <span class="fa fa-icon fa-external-link"
```

Comment:
Why not use directly oui-kit icons like `oui-icon-external_link` ?

Reply:
It doesn't not behave correctly

Reply:
No font-awesome icons